group_serviceaccounts.go

```diff
@@ -62,8 +62,9 @@ func (s *GroupsService) CreateServiceAccount(gid interface{}, options ...Request
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/groups.html#create-personal-access-token-for-service-account-user
 type CreateServiceAccountPersonalAccessTokenOptions struct {
-	Scopes *[]string `url:"scopes,omitempty" json:"scopes,omitempty"`
-	Name   *string   `url:"name,omitempty" json:"name,omitempty"`
+	Scopes    *[]string `url:"scopes,omitempty" json:"scopes,omitempty"`
+	Name      *string   `url:"name,omitempty" json:"name,omitempty"`
+	ExpiresAt *string   `url:"expires_at,omitempty" json:"expires_at,omitempty"`
```

Comment:
```suggestion
	ExpiresAt *ISOTime   `url:"expires_at,omitempty" json:"expires_at,omitempty"`
```

I guess this should be `ISOTime`?

Reply:
Updated 👍